### PR TITLE
Updated animal description for Cat

### PR DESCRIPTION
**Pull request by paul555asha**

### CODE REVIEW

**Comment by leftblank0** [comment]
I'll take a look at this one

**Comment by leftblank0** [comment]
@paul555asha 
Great job! Can you please also do the following?

1. Update the age of the pet to something that isn't 50
2. Replace 2.jpg in \AnimalAdoption.Web.Portal\wwwroot\images\pets with a picture of the cat.

File: AnimalAdoption.Common.Logic/AnimalService.cs [hunk 1]
```diff
@@ -8,7 +8,7 @@ public class AnimalService
     {
         public Animal[] ListAnimals => new Animal[] {
             new Animal { Id = 1, Name = "Sedi", Age = 50, Description = "Soft natured" },
-            new Animal { Id = 2, Name = "Metamorph", Age = 50, Description = "Under a lot of pressure" },
+            new Animal { Id = 2, Name = "Cat", Age = 50, Description = "Fluffy Cat" },
```
Comment (leftblank0) [inline AnimalAdoption.Common.Logic/AnimalService.cs:11]: please update 50 to a good age for a cat

**Review by vinayrvarma — APPROVED** [review]
Looks good. Its fluffy

**Pull request merged**